packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -337,20 +351,6 @@ export class ResourceService {
       throw new Error(INVALID_DELETE_PROJECT_CONFIGURATION);
     }
 
-    const gitRepo = await this.prisma.gitRepository.findUnique({
```

Comment:
you can still delete the gitRepo record in case `override` is true - knowing that the resource we delete is not the  projectConfiguration